HW1/P4/P4.txt

```diff
@@ -1 +1,14 @@
 # Your discussion here
+
+The representation of my graph is an adjacency list for each character. Each entry in the RDD is of the following format:
+  (characterName, [a list of adjacent characterNames])
+
+Character Name    |   Number of nodes touched    | Diameter
+-----------------------------------------------------------
+CAPTAIN AMERICA   |               6408           |    3
```

Comment:
I think you did not strip the white spaces in the character name. If you did that, the answer would be 6403 instead of 6408.